packages/pcre/pcre.7.1.6/files/pcre.install

```diff
@@ -0,0 +1,6 @@
+etc: [
+  "setup.ml"
+  "setup.data"
+  "setup.log"
+  "_oasis_remove_.ml"
+]
```

Comment:
This is odd -- do you really want to install `setup.log` and `setup.data` into `opam/etc`?